core/src/Kokkos_Cuda.hpp

```diff
@@ -266,11 +266,19 @@ class Cuda {
   inline Impl::CudaInternal* impl_internal_space_instance() const {
     return m_space_instance;
   }
+  uint32_t impl_instance_id() const noexcept { return 0; }
```

Comment:
You can change later whenever you actually know where to get the instance ids from.